lib/node_modules/@stdlib/lapack/base/dlange/lib/base.js

```diff
@@ -0,0 +1,362 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var dlassq = require( '@stdlib/lapack/base/dlassq' ).ndarray;
+var isRowMajor = require( '@stdlib/ndarray/base/assert/is-row-major' );
+var loopOrder = require( '@stdlib/ndarray/base/nullary-loop-interchange-order' );
+var dasum = require( '@stdlib/blas/base/dasum' ).ndarray;
+var Float64Array = require( '@stdlib/array/float64' );
+var min = require( '@stdlib/math/base/special/min' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var abs = require( '@stdlib/math/base/special/abs' );
+
+
+// FUNCTIONS //
+
+/**
+* Returns the value of the one norm of a real matrix `A`.
+*
+* @private
+* @param {NonNegativeInteger} M - number of rows in `A`
+* @param {NonNegativeInteger} N - number of columns in `A`
+* @param {Float64Array} A - input array
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {NonNegativeInteger} offsetA - starting index of `A`
+* @param {Float64Array} work - work array, should have `N` indexed elements if row-major layout is used
+* @param {integer} strideWork - stride length of `work`
+* @param {NonNegativeInteger} offsetWork - starting index of `work`
+* @returns {number} required norm value
```

Comment:
```suggestion
* @returns {number} result
```

"required" norm value is an odd description. Fine to just use `result` here and below.